src/components/button/Button.ts

```diff
@@ -1,23 +1,22 @@
 import Block from '../../utils/Block';
 import tpl from './tpl.hbs';
+import * as styles from './style.module.css';
 
 type ButtonProps = {
   label: string;
-  attr: Record<string, any>,
+  type? : string;
```

Comment:
тут можно конкретизировать тип, используя union.
Сейчас мы можем положить туда любую строку и TS не выдаст ошибку, значит мы словим ее в рантайме. Если мы будем использовать union и случайно сделаем опечатку в строке, то получим предупреждение и отловим ошибку на этапе написания кода
```suggestion
  type? : 'submit' | 'button' | 'reset';
```

Reply:
Сделала